src/components/UserRequests/requests.js

```diff
@@ -267,6 +314,36 @@ export default class Requests extends Component {
                             ) : (
                               <></>
                             )}
+                            {this.state.inProgressRequests.length > 0 ? (
+                              <tbody>
```

Comment:
can't we make this a common component for each type of request

Reply:
Yes we can I will modify that